zebra-rpc/src/methods.rs

```diff
@@ -1936,11 +1935,10 @@ where
 
         let client_long_poll_id = parameters.as_ref().and_then(|params| params.long_poll_id);
 
-        // - One-off checks
-
-        // Check config and parameters.
-        // These checks always have the same result during long polling.
-        let miner_address = get_block_template::check_miner_address(miner_address)?;
+        let miner_address = self
+            .gbt
+            .miner_address()
+            .ok_or_misc_error("miner_address not configured")?;
```

Comment:
Optional/Nitpick/Subjective: The error message before seemed a bit more descriptive, could we add that to the current message?

```suggestion
        let miner_address = self.gbt.miner_address().ok_or_misc_error(
            "miner_address not configured, set `mining.miner_address` in `zebrad.toml` to a transparent address",
        )?;
```

Reply:
I'm happy to merge these suggestions, but I can't think of an easy way to do so since the PR is merged.